.github/workflows/performance.yml

```
@@ -0,0 +1,211 @@
+      run: |
+        flutter pub get
+        flutter doctor -v
+        
+    - name: Create test results directory
+      run: mkdir -p test_results/performance
+      
+    - name: Run performance tests
+      run: |
+        dart test/performance/ci_runner.dart
+      continue-on-error: true
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)